src/DotNetWorker.Core/Converters/Converter/ConversionResult.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// A type representing the result of function input conversion operation.
+    /// </summary>
+    public readonly struct ConversionResult
+    {
+        /// <summary>
+        /// Gets a value indicating whether the conversion operation was successful or not.
+        /// </summary>
+        public bool IsSuccess { get; }
+
+        /// <summary>
+        /// Gets the value produced from the conversion operation if it was succesful.
+        /// </summary>
+        public object? Model {get;}
+
+        /// <summary>
+        /// Creates a new <see cref="ConversionResult"/> instance.
+        /// </summary>
+        /// <param name="isSuccess">Indicates the conversion operation was successful or not</param>
+        /// <param name="model">The value produced from the successful conversion.</param>
+        public ConversionResult(bool isSuccess, object? model)
+        {
+            IsSuccess = isSuccess;
+            Model = model;
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="ConversionResult"/> instance.
+        /// </summary>
+        public ConversionResult(object model)
```

Comment:
Curious when this instance is used and why the default for success is true?

Reply:
Good catch. 😄 This is unused and now removed.